public/js/studentController.js

```diff
@@ -4,10 +4,13 @@
     function getStudentById(id) {
         $.ajax({
             method: "GET",
-            url: "/api/students/" + id
+            url: "api/student/" + id
```

Comment:
Why have you change students to student?? Please take a look at [this good practices](http://www.restapitutorial.com/lessons/restquicktips.html), read "Provide Sensible Resource Names" section.